### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.servicefabric.models.ValidationFailedChaosEvent
  fullName: azure.servicefabric.models.ValidationFailedChaosEvent
  inheritance:
  - inheritance:
    - inheritance:
      - type: builtins.object
      type: msrest.serialization.Model
    type: azure.servicefabric.models.chaos_event.ChaosEvent
  langs:
  - python
  module: azure.servicefabric.models
  name: ValidationFailedChaosEvent
  source:
    id: ValidationFailedChaosEvent
    path: azure-servicefabric\azure\servicefabric\models\validation_failed_chaos_event.py
    remote:
      branch: master
      path: azure-servicefabric\azure\servicefabric\models\validation_failed_chaos_event.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 15
  summary: 'Chaos event corresponding to a failure during validation.

    '
  syntax:
    content: ValidationFailedChaosEvent(time_stamp_utc, reason=None)
    parameters:
    - description: ''
      id: time_stamp_utc
      type:
      - datetime
    - description: 'Polymorphic Discriminator

        '
      id: kind
      type:
      - str
    - description: ''
      id: reason
      type:
      - str
  type: class
  uid: azure.servicefabric.models.ValidationFailedChaosEvent
references: []
